pkg/dd/dashboard.go

```diff
@@ -1,14 +1,49 @@
 package dd
 
 import (
+	"fmt"
+
+	aw "github.com/deanishe/awgo"
 	"gopkg.in/zorkian/go-datadog-api.v2"
 )
 
-func QueryDashboard(client *datadog.Client) ([]datadog.DashboardLite, error) {
-	dashboards, err := client.GetDashboards()
-	if err != nil {
-		return nil, err
+type Dashboard struct {
+	DatadogWorkflowInterface
+	client    *datadog.Client
+	cacheName string
+	wf        *aw.Workflow
+}
+
+func NewDashboard(client *datadog.Client, wf *aw.Workflow) Dashboard {
+	return Dashboard{
+		cacheName: dashboardCacheName(),
+		client:    client,
+		wf:        wf,
+	}
+}
+
+func (d *Dashboard) ListDashboards() error {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method Dashboard.ListDashboards should have comment or be unexported](https://app.codacy.com/manual/nekottyo/alfred-datadog-workflow/pullRequest?prid=5421329)